docs/docs/themes/using-a-gatsby-theme.md

```diff
@@ -2,71 +2,55 @@
 title: Using a Gatsby Theme
 ---
 
-The quickest way to get started using a Gatsby theme is to use a starter that's configured to use the theme.
+While you can [get started quickly with a Gatsby theme starter](/docs/themes/getting-started/), you can also install a Gatsby theme directly to an existing Gatsby site. Gatsby themes are plugins, so we [use them just like any other Gatsby plugin](/docs/using-a-plugin-in-your-site/).
```

Comment:
Updating to follow the [Gatsby Style Guide](https://www.gatsbyjs.org/contributing/gatsby-style-guide/) for pronoun usage, and to omit the word "just".
```suggestion
While you can [get started quickly with a Gatsby theme starter](/docs/themes/getting-started/), you can also install a Gatsby theme directly to an existing Gatsby site. Gatsby themes are plugins, so you can [install and use them like any other Gatsby plugin](/docs/using-a-plugin-in-your-site/).
```